.github/ISSUE_TEMPLATE/new-hub.yml

```diff
@@ -0,0 +1,164 @@
+name: 💻  New Hub
+description: Creating a new hub for 2i2c to operate
+title: "New Hub: <Hub name>"
+labels: ["type: hub"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Ask for a new 2i2c JupyterHub!
+        
+        This template is designed to help you provide information about your 2i2c Hub so that a 2i2c Engineer can set it up properly.
+        Answer the sections below as best you can.
+        We can update information as we learn more.
```

Comment:
While I generally approve of line break after every period, unfortunately the issue form doesn't render this as we've come to expect with Markdown. This bugs me on a personal level and shouldn't block this PR, but just raising it in case I'm not alone in my annoyance 😆